### PR TITLE
fix golint in volume fs

### PR DESCRIPTION
**Pull request by LexCC**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Fix golint failures in pkg/volume/util/fs
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
None
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Review by 0xHazard — APPROVED** [review]

**Comment by dims** [comment]
/approve (for hack/)

@JieJhih please ping assignees in #sig-storage channel for review in a few days.

File: pkg/volume/util/fs/fs.go [hunk 1]
```diff
@@ -30,9 +30,9 @@ import (
 	"k8s.io/kubernetes/pkg/volume/util/fsquota"
 )
 
-// FsInfo linux returns (available bytes, byte capacity, byte usage, total inodes, inodes free, inode usage, error)
+// Info linux returns (available bytes, byte capacity, byte usage, total inodes, inodes free, inode usage, error)
 // for the filesystem that path resides upon.
-func FsInfo(path string) (int64, int64, int64, int64, int64, int64, error) {
+func Info(path string) (int64, int64, int64, int64, int64, int64, error) {
```
Comment (xing-yang) [inline pkg/volume/util/fs/fs.go:35]: Do you know why the func name "FsInfo" is not allowed by golint?

Comment (xing-yang) [reply pkg/volume/util/fs/fs.go:35]: Got it.
```
pkg/volume/util/fs/fs.go:35:6: func name will be used as fs.FsInfo by other packages, and that stutters; consider calling this Info
```

**Pull request merged**